titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 4_
```python
import seaborn as sns
import matplotlib.pyplot as plt

# Load the Titanic dataset
```
load titanci review comment 1

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
1